plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/ideMaven/MavenRunnerUtils.kt

```diff
@@ -66,6 +67,8 @@ fun runMavenCopyCommands(sourceFolder: File, buildlogBuilder: StringBuilder, log
         val transformMvnRunner = TransformMavenRunner(project)
         val mvnSettings = MavenRunner.getInstance(project).settings.clone() // clone required to avoid editing user settings
 
+        if (ProjectRootManager.getInstance(project).projectSdk == null) return MavenCopyCommandsResult.NoJdk
```

Comment:
it is possible for the project SDK to be null, but for a module SDK to be set.
additionally, the maven runner itself can be configured to use a specific SDK

Reply:
Good points. Addressed by: checking module-level SDK first, then falling back to project SDK. If both are null, notify the user, otherwise, configure MavenRunner to use the SDK found.

Reply:
This change will require some minor updates to our Strings and docs too.

Reply:
Strings and docs are now both updated.